components/vf-box/vf-box.scss

```diff
@@ -57,7 +74,9 @@
   .vf-content & a,
   a {
     color: currentColor;
+    position: relative;
     text-decoration: underline;
+    z-index: 1984;
```

Comment:
Should use the design token z-indexes for these?

Reply:
I was thinking that when I quickly type it. Will update